### PR TITLE
Python 3 and Django migrations (Django 1.7)

### PR DESCRIPTION
**Pull request by driesdesmet**

### CODE REVIEW

File: cmsplugin_randomquote/models.py [hunk 1]
```diff
@@ -1,11 +1,13 @@
+from __future__ import unicode_literals
 from django.db import models
 from django.utils.translation import ugettext_lazy as _
+from django.utils.encoding import python_2_unicode_compatible
 
-
+@python_2_unicode_compatible
```
Comment (dbrgn) [inline cmsplugin_randomquote/models.py]: That's very nice, didn't know about this decorator yet.

**Comment by dbrgn** [comment]
Do you know of a way to denote that South 1.0 is required to migrate this project?

**Comment by dbrgn** [comment]
Also, this PR now includes the merge commit. Could you rebase against the current master?

You can do it this way, in case you're not too familiar with git rebase:

```
git remote add upstream https://github.com/dbrgn/cmsplugin-randomquote/
git fetch upstream
git rebase upstream master
```

Or if you want to work around the rebasing:

```
git reset --hard upstream master
git cherry-pick a177c24
git cherry-pick fd54d3b
```

You can verify the result with `git log` and then force-push to your branch.

**Comment by driesdesmet** [comment]
About South 1.0: no, if you don't want to make it a dependancy, I guess the only way is to document it well in the README. But I can see why this is a bit of a hurdle if you want to maintain < Django1.7 compatibility. Maybe it's time for another release and explicitly document that people who have older Django versions are better of installing and older version.

Let me try that rebasing thing.

**Comment by dbrgn** [comment]
> Maybe it's time for another release and explicitly document that people who have older Django versions are better of installing and older version.

True, people wanting to support an older version can still use the current release if they wish :)

**Comment by driesdesmet** [comment]
Think I've managed. I got a bit confused because my branch was named 'urga' and not 'master'.

**Comment by dbrgn** [comment]
Thanks!

**Comment by dbrgn** [comment]
Let me know when you're done with all your changes, then we can think about a new release.

**Pull request merged**